skbio/stats/composition.py

```diff
@@ -608,10 +612,274 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0-dev")
+def ancom(matrix, categories,
```

Comment:
`grouping` or `groups` may be a better name than `categories` because the docs refer to groups. Also, `skbio.stats.distance.anosim/permanova` use `grouping` for this vector to match R vegan interface.